.github/workflows/publish-release.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Bundle library
         run: yarn build
       - name: Create standalone binary
-        run: yarn dist-standalone -t node14-linux-x64 -o datadog-ci_linux-x64
+        run: yarn dist-standalone -t node18-linux-x64 -o datadog-ci_linux-x64
```

Comment:
For future reference, this will increase the binary size:

| File name               | Size before | Size after |
| ----------------------- | ----------- | ---------- |
| `datadog-ci_win-x64`      | 53.5 MB     | 60 MB      |
| `datadog-ci_darwin-arm64` | 61 MB       | 68 MB      |
| `datadog-ci_darwin-x64`   | 65.7 MB     | 73 MB      |
| `datadog-ci_linux-arm64`  | 58.3 MB     | 66 MB      |
| `datadog-ci_linux-x64`    | 59.6 MB     | 68 MB      |